adagrams/game.py

```diff
@@ -1,11 +1,156 @@
+from operator import le
+import random
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+SCORE_CHART =  {
+    'A': 1, 
+    'B': 3, 
+    'C': 3, 
+    'D': 2, 
+    'E': 1, 
+    'F': 4, 
+    'G': 2, 
+    'H': 4, 
+    'I': 1, 
+    'J': 8, 
+    'K': 5, 
+    'L': 1, 
+    'M': 3, 
+    'N': 1, 
+    'O': 1, 
+    'P': 3, 
+    'Q': 10, 
+    'R': 1, 
+    'S': 1, 
+    'T': 1, 
+    'U': 1, 
+    'V': 4, 
+    'W': 4, 
+    'X': 8, 
+    'Y': 4, 
+    'Z': 10
+}
+def create_bag_of_letters():
+    """
+    Returns an array containing all the letters of the "LETTER_POOL" dictionary 
+    according to the frequency associated with each letter.
+    Ex: Since the value of 'A' is 9, the first 9 elements of this list will be 'A' 
+    appearing 9 times, and so on.
+    bag_of_letters = ['A', 'A', ..., 'B', ...]
+    """
+    bag_of_letters = []
+    for letter in LETTER_POOL:
+        number_of_letters = LETTER_POOL[letter]
+        for i in range(number_of_letters):
+            bag_of_letters.append(letter)
+    return bag_of_letters
+
 def draw_letters():
-    pass
+    hand_of_letters = [] # to be filled with 10 one-letter strings
+    bag_of_letters = create_bag_of_letters()
+   
+    for i in range(10):
+        #size_of_letter_bag = len(bag_of_letters)
```

Comment:
Minor style preference: Building on to that, I would normally remove commented code just to keep things clean.